### PR TITLE
Added a couple more examples like calculations, string counting, file…

### PR DESCRIPTION
**Pull request by ozombo**
What does this PR do?
Creates readme file
Added couple more functions which include calculations, files and directories, string counting and much more.

Where does reviewer start?
have a look at the new .go files created

### CODE REVIEW

File: countwords.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			counts[word] += 1
+		} else {
+			counts[word] = 1
+		}
+	}
+	return counts
+}
+
+func main() {
+	strLine := "ozombo unicodedeveloper node ichtrojan codebeast vscode ozombo ichtrojan twitter codebeast ozombo"
```
Comment (ichtrojan) [inline countwords.go]: we should make the variable named `sentence` and the content should be an actual sentence like `The quick brown fox jumps over the lazy dog`

File: itemexists.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+)
+
+//check if any item exists or not
+func main() {
+	var strSlice = []string{"ozombo", "twilio", "unicodedeveloper", "amycruz", "codebeast"}
```
Comment (ichtrojan) [inline itemexists.go]: The contents should contain actual words not the Monica or names of people. To make this example practical use fruits

```go
func main () {
    fruits := [string{"apple", "banana", "mango"}
}
```

refactor the whole file to fit this

File: minimum.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+// find mix or max number between two defined numbers
+
+func main() {
+	a := math.Min(100, 20)
+	fmt.Println(a)
+
+	b := math.Max(100, 20)
+	fmt.Println(b)
+}
```
Comment (ichtrojan) [inline minimum.go]: I feel this file should be renamed to `math.go` and should contain frequently used math methods in go

File: oddoreven.go [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+//check odd or even number
+func main() {
+	fmt.Print("Enter number : ")
+	var n int
+	fmt.Scanln(&n)
+	/*  Conditional Statement if .... else ........     */
```
Comment (ichtrojan) [inline oddoreven.go]: This comment is not necessary

File: oddoreven.go [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+//check odd or even number
+func main() {
+	fmt.Print("Enter number : ")
+	var n int
```
Comment (ichtrojan) [inline oddoreven.go:12]: This should be declared globally outside main

File: permutations.go [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    "fmt"
+    "os"
+    "strconv"
+)
+  
+func rangeSlice(start, stop int) []int {
+    if start > stop {
+        panic("Slice ends before it started")
+    }
+    xs := make([]int, stop-start)
```
Comment (ichtrojan) [inline permutations.go]: we should used meaningful variable names `xs` is not clear to me

File: switch.go [hunk 7]
```diff
@@ -2,18 +2,18 @@ package main
...
-      fmt.Println("💰💰💰💰💰💰💰")
+	case "money":
+		fmt.Println("💰💰💰💰💰💰💰")
 
-    default:
-      fmt.Println("I cannot give you what you desire")
-  }
+	default:
+		fmt.Println("I cannot give you what you desire")
+	}
```
Comment (ichtrojan) [inline switch.go:18]: Also, I feel the whitespacing is too much

File: README.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Various GO Lang examples to help you get a better understanding of the langauage
+
+To test any of the examples:
+
+go run filename.go
+
```
Comment (ichtrojan) [inline README.md:6]: I think we should get rid of the comments in the code then explain what each file does in the readme and how

File: switch.go (hunk 7 above)
Comment (ozombo) [reply switch.go:18]: Go formatting

File: README.md (hunk 8 above)
Comment (ozombo) [reply README.md:6]: Smart idea but then comments might help readers get better insights.

File: countwords.go (hunk 1 above)
Comment (ichtrojan) [reply countwords.go]: @ozombo

File: itemexists.go (hunk 2 above)
Comment (ichtrojan) [reply itemexists.go]: @ozombo

File: minimum.go (hunk 3 above)
Comment (ichtrojan) [reply minimum.go]: @ozombo

File: oddoreven.go (hunk 4 above)
Comment (ichtrojan) [reply oddoreven.go]: @ozombo

File: minimum.go (hunk 3 above)
Comment (ozombo) [reply minimum.go]: I'd update this.

Comment (ozombo) [reply minimum.go]: Take a look @ichtrojan

File: oddoreven.go (hunk 4 above)
Comment (ozombo) [reply oddoreven.go]: Oh ok

Comment (ozombo) [reply oddoreven.go]: @ichtrojan removed

**Pull request merged**